prog/weaver/http.go

```diff
@@ -161,7 +161,12 @@ func HandleHTTP(muxRouter *mux.Router, version string, router *weave.Router, all
 	}
 	muxRouter.Methods("GET").Path("/report").Headers("Accept", "application/json").HandlerFunc(
 		func(w http.ResponseWriter, r *http.Request) {
-			json, _ := json.MarshalIndent(status(), "", "    ")
+			json, err := json.MarshalIndent(status(), "", "    ")
+			if err != nil {
+				Log.Error(err)
```

Comment:
In the event that the marshalling fails, you see something like this:

````
$ weave report
json: unsupported type: odp.FlowKeys
````

This should only ever happen when a developer is extending the report JSON however.